src/test/java/com/google/firebase/auth/FirebaseUserManagerTest.java

```diff
@@ -437,19 +437,23 @@ public void testGetTenant() throws Exception {
     Tenant tenant = FirebaseAuth.getInstance().getTenantManager().getTenantAsync("TENANT_1").get();
     checkTenant(tenant, "TENANT_1");
     checkRequestHeaders(interceptor);
+    checkUrl(interceptor,
+        "https://identitytoolkit.googleapis.com/v2/projects/test-project-id/tenants/TENANT_1");
```

Comment:
Nit: Define the URL prefix (up to the /tenants portion) in a constant.

Reply:
Done.